telemetry/definitions/commonDefinitions.json

```diff
@@ -87,6 +87,16 @@
             "type": "string",
             "description": "The name of the AWS service acted on. These values come from the AWS SDK. To find them in the JAVA SDK search for SERVICE_NAME in each service client, or look for serviceId in metadata in the service2.json"
         },
+        {
+            "name": "requestServiceId",
+            "type": "string",
+            "description": "The service identifier associated with a request. Values should have the same shape as `serviceType`. This is distinct from `serviceType` which describes the originator of the request rather than the reqeust itself."
+        },
+        {
+            "name": "requestId",
```

Comment:
We could join the request ID + service ID into `${serviceId}:${requestId}` to be more defensive but now we're assuming a specific shape whereas keeping them separate allows another field to act as a disambiguator.